src/components/layout/Footer.less

```diff
@@ -4,30 +4,72 @@
   text-align: center;
   vertical-align: middle;
   min-height: 100px;
+  padding: 0 20px;
 }
 
+
+/* stylelint-disable */
+@media screen and (max-width: 1090px) {
+  .footer {
+    height: unset !important; // last resort override
+  }
+}
+/* stylelint-enable */
+
 .footer__nav {
   background-color: var(--g3-color__bg-coal);
-  width: 1220px;
   color: white;
-  vertical-align: middle;
-  margin: auto;
-  text-align: left;  
+  text-align: left;
+  display: flex;
+  justify-content: space-between;
+}
+
+@media screen and (max-width: 1090px) {
+  .footer__nav {
+    padding: 20px 0;
+    flex-flow: column;
+  }
 }
 
 .footer__version-area {
   display: flex;
-  float: left;
-  width: 720px;
-  height: 100px;
+  flex-basis: 50%;
+}
+
+@media screen and (max-width: 1090px) {
+  .footer__version-area {
+    width: 100%;
+    justify-content: stretch;
+    text-align: center;
+  }
+}
+
+@media screen and (max-width: 820px) {
+  .footer__version-area {
+    width: 100%;
```

Comment:
This line of CSS appears to be overridden by some other CSS that makes the width 600px, resulting in horizontal scroll on small screens. When I made this 
width: 100%; !important
the problem was fixed :P